README.md

```diff
@@ -18,23 +18,24 @@ the request using `RequestId`.
 
 ```rust
 use actix_request_identifier::{RequestId, RequestIdentifier};
+use actix_web::{get, App, HttpServer, Responder};
 
 #[get("/")]
-async fn index(id: RequestId) -> impl Responder {
-    HttpResponse::Ok().body(format!("{}", id.as_str()))
+async fn show_request_id(id: RequestId) -> impl Responder {
+    format!("{}", id.as_str())
 }
 
-#[actix_rt::main]
-async fn main() {
-    HttpServer::new(move || {
+#[actix_web::main] // or #[tokio::main]
+async fn main() -> std::io::Result<()> {
+    let http_server = HttpServer::new(|| {
```

Comment:
Shouldn't the result of `...run().await` be returned from `main`? I think the `let http_server =` can be removed, or `http_server` should be returned in the end.